packages/E2ETest/run_wdio.js

```diff
@@ -100,20 +114,42 @@ function parseLogs() {
   return names;
 }
 
+function PrintFailedTests(ft) {
+  for (let i = 0; i < Object.keys(ft).length; i++) {
+    const key = Object.keys(ft)[i];
+    console.log(chalk.redBright(key));
+    console.log(
+      '  ',
+      chalk.underline('testcase'),
+      chalk.bold(ft[key].testcase)
+    );
+    console.log('  ', chalk.underline('error'), chalk.bold(ft[key].error));
+    console.log('  ', chalk.underline('stack'));
+    console.log(ft[key].stack);
+  }
+}
+
 function Process(code) {
   const failedTests = parseLogs();
   for (let i = 0; i < failedTests.length; i++) {
-    console.log(`Failed test: ${failedTests[i]}`);
+    console.log('Failed tests: ');
+    PrintFailedTests(failedTests[i]);
   }
   process.exit(code);
 }
 
-wdio.run().then(
-  code => {
-    Process(code);
-  },
-  error => {
-    console.error('Launcher failed to start the test', error.stacktrace);
-    process.exit(1);
-  }
-);
+function RunWdio() {
```

Comment:
nit: JS functions should be lowercase (this is inconsistent in this file it looks like)

Reply:
Today I learned that you can't have a function called `process` all in lowercase. It won't error out with anything meaningful, but your process.argv evaluation won't work because of this "override". 

I am not a fan of JavaScript.

Reply:
It will produce a compile time error and show up red in your IDE if you're using Typescript.

But yeah, it's a global object in Node.